relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -426,17 +430,24 @@ fn extract_transaction_metrics_inner(
         tags_with_satisfaction.clone(),
     ));
 
+    let mut root_counter_tags = BTreeMap::new();
+    if let Some(name) = transaction_from_dsc {
+        root_counter_tags.insert("transaction".to_owned(), name.to_owned());
+    }
+    let decision = match sampling_result {
+        SamplingResult::Keep => "keep".to_owned(),
+        SamplingResult::Drop(_) => "drop".to_owned(),
+    };
+    root_counter_tags.insert("decision".to_owned(), decision);
```

Comment:
Does it make sense to implement `ToString` in `SamplingResult`, and directly `sampling_result.to_string()`?

Reply:
In this case I would likely not use it, because if we look at `SamplingResult` in isolation we would also render the rule IDs from `Drop` (see the `Display` implementation of `MatchedRuleIds`).